dc/core/resources/BorderRectangle.qml

```diff
@@ -39,7 +39,7 @@ Item {
         width: Style.borderWidth
         height: Style.borderWidth
         radius: Style.borderWidth
-        color: Style.inactiveBorderColor
+        color: "#00000000"
```

Comment:
There was a duplication of the "color: Style.inactiveBorderColor" in here and in the "inactive" state (which is also set by default, thus maybe your confusion), but at least it was consistent. Just tweak the states until it works as desired ;-)